src/components/Todo/TodoInfo.tsx

```diff
@@ -0,0 +1,57 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  handleChangeStatus: (todoId: number, status: boolean) => void,
+  handleDeleteTodo: (todoId: number) => void,
+};
+
+export const TodoInfo: React.FC<Props> = ({
+  todo,
+  handleChangeStatus,
+  handleDeleteTodo,
+}) => {
+  const { id, title, completed } = todo;
+
+  return (
+    <div
+      data-cy="Todo"
+      className={classNames(
+        'todo',
+        {
+          completed,
+        },
```

Comment:
correct using of cn

Reply:
Here something is wrong with class usage. Write how it should be [here](https://www.npmjs.com/package/classnames)

```suggestion
      className={classNames(
        'todo',
        { `some class`: condition },
```